engine/install/debian.md

```diff
@@ -65,24 +65,23 @@ You can install Docker Engine in different ways, depending on your needs:
 - [Install it manually](#install-from-a-package) and manage upgrades manually.
 
 - Use a [convenience scripts](#install-using-the-convenience-script). Only
-  recommended for testing and development environments. This is the only
-  approach available for Raspbian.
+  recommended for testing and development environments.
 
 ### Install using the apt repository {#install-using-the-repository}
 
 Before you install Docker Engine for the first time on a new host machine, you
 need to set up the Docker repository. Afterward, you can install and update
 Docker from the repository.
 
-> **Raspbian users can't use this method.**
->
-> For Raspbian, installing using the repository isn't yet supported. You must
-> instead use the [convenience script](#install-using-the-convenience-script).
-
 #### Set up the repository
 
 {% assign download-url-base = "https://download.docker.com/linux/debian" %}
 
+> **Note**
+>
+> If you're using Raspbian, skip ahead to
+> [Set up repository (Raspbian)](#set-up-the-repository-raspbian).
```

Comment:
I'm still on the fence what'd be easier to maintain;

- combined "raspbian" / "debian" page
- or just duplicate "debian" and make the changes for "raspbian"

Latter one

- prevents some amount of "navigate left/right/up/down" to pick the right instructions, and users can just consume the whole page as usual
- when we need to update things in debian/ubuntu/raspbian, usually those changes are identical across those, and it's easier to do a side-by-side diff to synchronise those changes.

wdyt?

Reply:
I guess it would also help discovery; as currently users need to know to look under "Debian" if they want to install "Raspbian"
<img width="284" alt="Screenshot 2023-05-11 at 14 25 23" src="https://github.com/docker/docs/assets/1804568/58367799-5dd1-4ffc-bc4e-9394c9b277a2">